src/main/java/com/tudu/todoapp/rest/controllers/UserController.java

```diff
@@ -20,12 +20,31 @@ public ResponseEntity<?> createUser(@RequestBody User user) {
         return new ResponseEntity<>("created yea", HttpStatus.OK);
     }
 
+
+
+
+
     @PutMapping("/{id}")
     public ResponseEntity<User> updateUser(@PathVariable Long id, @RequestBody User newData) {
         return new ResponseEntity<>(userService.updateUser(id, newData), HttpStatus.OK);
     }
 
 
+    @PostMapping("/save")
```

Comment:
This whole method is not needed. We have a `createUser` method in this controller that does the same. You don't have to pass anything else, just a User DTO or a whole user object, spring handles everything else for you.

Please, remove this endpoint as it's redundant.